pkg/synapse/synapse.go

```diff
@@ -5,30 +5,39 @@ import (
 	"encoding/json"
 	"fmt"
 	"sync"
+	"time"
 
 	"github.com/LambdaTest/synapse/pkg/core"
 	"github.com/LambdaTest/synapse/pkg/global"
 	"github.com/LambdaTest/synapse/pkg/lumber"
 	"github.com/LambdaTest/synapse/pkg/utils"
 	"github.com/denisbrodbeck/machineid"
 	"github.com/gorilla/websocket"
+	"github.com/lestrrat-go/backoff"
 	"github.com/spf13/viper"
 )
 
 // All constant related to synapse
 const (
-	Repo    = "repo"
-	BuildID = "build-id"
-	JobID   = "job-id"
-	Mode    = "mode"
-	ID      = "id"
+	Repo                   = "repo"
+	BuildID                = "build-id"
+	JobID                  = "job-id"
+	Mode                   = "mode"
```

Comment:
what is mode here ?

Reply:
Mode defines the type of job , namely parsing , execution , coverage